defaults/main.yml

```diff
@@ -16,6 +18,7 @@ openwisp2_django_loci_pip: false
 openwisp2_netjsonconfig_pip: false
 openwisp2_network_topology_pip: false
 openwisp2_firmware_upgrader_pip: false
+openwisp2_monitoring_pip: true
```

Comment:
The default should be `false`.

@Purhan  we use `openwisp2_**_pip` variables to pass an URL so that pip installs that module from that URL. E.g. if you want to install development version of openwisp-monitoring, you would set

```yml
openwisp2_monitoring_pip: "https://github.com/openwisp/openwisp-monitoring/tarball/master"
```